pkg/metricsstore/metricsstore.go

```diff
@@ -52,6 +52,10 @@ type MetricsStore struct {
 	// ProxyXDSRequestCount counts XDS requests made by proxies
 	ProxyXDSRequestCount *prometheus.CounterVec
 
+	// ProxyMaxConnectionsRejected counts the number of proxy connections
+	// rejected due to the max connections limit being reached.
```

Comment:
nit: remove "." for consistency
```suggestion
	// rejected due to the max connections limit being reached
```